src/java.base/share/native/launcher/main.c

```diff
@@ -36,6 +36,13 @@
 
 #include <limits.h>
 
+// Windows doesn't have PATH_MAX. It's MAX_PATH instead.
+#ifdef _WIN32
```

Comment:
Thanks. Checking https://learn.microsoft.com/en-us/cpp/c-language/cpp-integer-limits?view=msvc-170, PATH_MAX is not defined for Windows limits.h.